modules/platforms/cpp/core/include/ignite/impl/operations.h

```diff
@@ -252,6 +252,40 @@ namespace ignite
             IGNITE_NO_COPY_ASSIGNMENT(InCacheLocalPeekOperation)
         };
 
+        /**
+         * Cache Invoke input operation.
+         */
+        template<typename T1, typename T2>
+        class InCacheInvokeOperation : public InputOperation
+        {
+        public:
+            /**
+             * Constructor.
+             *
+             * @param val1 First value.
+             * @param val2 Second value.
+             */
+            InCacheInvokeOperation(const T1& val1, const T2& val2) : val1(val1), val2(val2)
+            {
+                // No-op.
+            }
+
+            virtual void ProcessInput(ignite::impl::binary::BinaryWriterImpl& writer)
+            {
+                writer.WriteTopObject<T1>(val1);
+                writer.GetStream()->WriteInt64(0);
```

Comment:
You can just type `writer.WriteInt64(0);`, no need to get stream here.

Reply:
Good point, fixed.